merge_frame_num: 8 # 叠加历史帧数目，为整数，大于等于1
roi: # 判断是否发生碰撞区域(单位:m)
  x_min: 0
  x_max: 15
  y_min: -10
  y_max: 10
  z_max: 0.6 # 适应室内环境
  overlap_area_remove: # 激光雷达的重复视野区域会有标定误差带来的跳变点，因此使用以下角度参数忽略掉对应区域，只设置左边，右边的通过对称性去除
    angle_up_right: 21 # 左数第二个激光雷达与左数第三个激光雷达重叠区域中靠右的那条边的角度(单位：度)
    angle_up_left: 34 # 左数第二个激光雷达与左数第三个激光雷达重叠区域中靠左的那条边的角度(单位：度)
    angle_down_right: 76 # 左数第一个激光雷达与左数第二个激光雷达重叠区域中靠右的那条边的角度(单位：度)
threshold: # 跳变点的阈值判断，两个连续点的高度差大于height，且在xy平面的距离差小于dist则为跳变点
  height: 0.1 # 两个连续点的高度差阈值
  dist: 0.1 # 两个连续点的xy平面距离差阈值
max_z_height:  0.3 #0.1 # 超过这个高度的认为是无需处理的较高障碍物
max_z_map_pixel_scale: 2 # 最高高度图的分辨率（1米分max_z_map_pixel_scale个格子的意思，数值越大，分辨率越大）
